capstone/src/main/java/com/google/univiz/scorecard/URLProviderImpl.java

```diff
@@ -14,8 +15,16 @@
 import javax.inject.Inject;
 
 final class URLProviderImpl implements URLProvider {
+  private static final String FRONT_URL =
+      "https://api.data.gov/ed/collegescorecard/v1/schools.json?";
+  private static final String QUERY_TYPE_NAME = "school.name=";
+  private static final String QUERY_TYPE_ID = "id=";
+  private static final String PER_PAGE = "&per_page=";
+  private static final String FIELDS_PARAM = "&fields=";
+  private static final String SUGGESTION_PER_PAGE = "20";
   private final UnivizConfig univizConfig;
   private final Set<String> fields;
+  private final Set<String> fieldsSugg;
```

Comment:
Avoid acronyms in naming unless it's making the code more readable in some other way. Here there is no reason to use an acronym. Just rename to suggestionFields,

Reply:
ok thanks!